int32 Size
float32[] Measure
float32[] Actual
---
float32[] Matrix